apps/linode-marketplace-linuxgsm/provision.yml

```diff
@@ -0,0 +1,13 @@
+# linode ansible playbook
+
+# linodes
+- name: setting up environment 
+  hosts: localhost
+  connection: local
+  any_errors_fatal: true
+  user: root
+  vars_files:
+    - group_vars/linode/vars
+  tasks:
+    - name: generating sudo user and password 
+      import_tasks: ../linode_helpers/roles/sudouser/tasks/main.yml
```

Comment:
Probably, but it would need to be tested. Importing the atomic task is how we're doing it in all of the provison.yml files. 
https://github.com/akamai-compute-marketplace/marketplace-apps/blob/f1887d7234bf274b0d8003fe38d908c9d70d6c03/apps/linode-marketplace-odoo/provision.yml#L12